### PR TITLE
git-log: update examples

### PR DESCRIPTION
**Pull request by Alfriadox**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [ ] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [ ] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Comment by navarroaxel** [comment]
We're still in 9 @Alfriadox. Maybe you can get inspiration from https://github.com/tldr-pages/tldr/blob/master/pages/common/git-stash.md

We can merge `git log` and `git log -p` using:
```
git log [-p {{path/to/file_or_directory}}]
```

Maybe we should hear the opinion from another reviewer

**Comment by Alfriadox** [comment]
My bad I miscounted, will fix this evening

File: pages/common/git-log.md [hunk 1]
```diff
@@ -3,13 +3,9 @@
 > Show a history of commits.
 > More information: <https://git-scm.com/docs/git-log>.
 
-- Show the sequence of commits starting from the current one, in reverse chronological order:
+- Show the sequence of commits starting from the current one, in reverse chronological order (optionally specifying a path):
```
Comment (sbrl) [inline pages/common/git-log.md:6]: Specifying a path to what?

I'm a bit confused here. Also, the optional bit with the square brackets makes this example quite complex - especially for the 1st example on the page. For the beginner I think I'd prefer to keep a basic `git log` without any arguments as the first example.

Comment (Alfriadox) [reply pages/common/git-log.md]: Fixed I think -- Just removed the complex / confusing bits.

File: pages/common/git-log.md [hunk 2]
```diff
@@ -3,14 +3,10 @@
...
 > More information: <https://git-scm.com/docs/git-log>.
 
-- Show the sequence of commits starting from the current one, in reverse chronological order:
+- Show the sequence of commits starting from the current one, in reverse chronological order of the git repository in the current working directory:
 
 `git log`
 
-- Show the history of a particular file or directory, including differences:
-
-`git log -p {{path/to/file_or_directory}}`
```
Comment (navarroaxel) [inline pages/common/git-log.md:12]: Hey @sbrl, I think the `-p` argument is important and really useful to get it handy, what if we keep this example and we merge it with another like `--stat`. Or, option B, what if we remove this complex example:

```bash
git log --oneline --decorate --all --graph
```

Comment (navarroaxel) [inline pages/common/git-log.md:12]: Hey @sbrl, I think the `-p` argument is important and really useful to get it handy, what if we keep this example and we merge it with another like `--stat`. Or, option B, what if we remove this complex example:

```bash
git log --oneline --decorate --all --graph
```

Comment (navarroaxel) [reply pages/common/git-log.md:12]: @sbrl I merged the examples with `--graph` and `--oneline`. Could you please review?

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Looks ok, thanks @Alfriadox!

**Pull request merged**